README.md

```diff
@@ -69,16 +69,37 @@ For icons to appear, add the following icon font download to your page.
 ```
 
 ## Custom component styles
+The the styles of these components can be customized via sass mixins.
```

Comment:
Extra “the” here. (I’m excited about these changes, in case you’re wondering why I’m looking at this PR. 😂 I’m rewriting my site using AngularDart and Angular Components. 😄)